sycl/test/lit.cfg.py

```diff
@@ -56,7 +56,14 @@
     llvm_config.with_environment('CPATH', "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/", append_path=True)
     llvm_config.with_environment('DYLD_LIBRARY_PATH', config.sycl_libs_dir)
 
-llvm_config.with_environment('PATH', config.sycl_tools_dir, append_path=True)
+# propagate the environment variable OCL_ICD_FILANEMES to use proper runtime.
+if 'OCL_ICD_FILENAMES' in os.environ:
```

Comment:
Why do we need this? Can you commit it as a separate patch?

Reply:
It's merge glitch.
Removed.